lib/mongo/scrollable.rb

```diff
@@ -41,11 +41,14 @@ def scroll(cursor_or_type = nil, options = nil, &_block)
       # scroll
       if block_given?
         previous_cursor = nil
+        current_cursor = nil
         records.each do |record|
           previous_cursor ||= cursor_type.from_record(record, cursor_options.merge(type: :previous))
+          current_cursor ||= cursor_type.from_record(record, cursor_options.merge(include_current: true))
           iterator = Mongoid::Criteria::Scrollable::Iterator.new(
             previous_cursor: previous_cursor,
-            next_cursor: cursor_type.from_record(record, cursor_options)
+            next_cursor: cursor_type.from_record(record, cursor_options),
+              current_cursor: current_cursor
```

Comment:
Some rubocop will be needed here.

Reply:
I changed, thanks.

Rubocop show me 138 offenses with the config in .rubocop.yml :disappointed_relieved: